### ISSUE TITLE
Error in running MITETracker

### ISSUE THREAD

**Issue by dipsaha** [INTABiotechMJ/MITE-Tracker#3]
Hi,

I am trying yo install and run MITETracker using a virtualenviornment in Ubuntu 14.04.lts. But I am unable to run the MITETracker.py. I follwed the below steps:
1. First installed and created a virtualenviornment in home dir venv
2. virtualenv -p python3 venv
source venv/bin/activate
3. Installed and upgraded python in the venv dir
python --version
 Python 3.6.3
4. Cloned MITETracker from venv:
 git clone https://github.com/INTABiotechMJ/MITE-Tracker.git
5. pip install -r /home/icar/MITE-Tracker/requirements.txt
(This installed all required packages including Biopython, numpy and pandas)
6. then tried:
nohup python3 -m MITETracker -g /home/icar/Jute_genomes/LLWS01.1.fa -w 8 -j 524_MITEtracker_out &

but ended up with the following error:
(venv) icar@icar-crijaf:~$ nohup: ignoring input and appending output to ‘nohup.out’

The nohup output says:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/local/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/icar/MITE-Tracker/MITETracker.py", line 3, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'

KINDLY help me to fix the issue and run the MITETracker properly. I am novice in working with python.
Your help in solving the issue in simpler terms and step by step process will be highly appreciated.

Regards
Dip

**Comment by juancresc** [comment]
Hi,

when running, please make sure you're in the same directory as of MITE Tracker. 

Would be 

> cd /home/icar/MITE-Tracker

**Comment by juancresc** [comment]
I'll close it since no more updated have been reported

**Comment by Marwa-ZI** [comment]
Hi,
I have installed MITE Tracker.
When I tried the command line "python3 -m MITETracker -g /home/zidi/Documents/data/genome-b.tabaci/genome.fa -w 3 -j miteb-tab " from  the same directory as of MITE Tracker it showed an error : 

Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/zidi/Documents/mitetracker/MITE-Tracker/MITETracker.py", line 3, in <module>
    from Bio import SeqIO
ModuleNotFoundError: No module named 'Bio'
 
Please help me to fix the issue and run the MITETracker properly.

Thanks
Marwa

**Comment by juancresc** [comment]
Hello Marwa.

You need to install biopython first

`pip install biopython`

I'd suggest that you use a virtual environment or conda for that.

**Comment by juancrescente** [comment]
Hello Marwa.

You need to install biopython first

`pip install biopython`

I'd suggest that you use a virtual environment or conda for that.

**Comment by Marwa-ZI** [comment]
Thank you 
it's done
